app/assets/stylesheets/upvote/posts.css.scss

```diff
@@ -6,6 +6,25 @@ h3.no-posts {
 $vote_container_size: 45px;
 $vote_container_margin: 12px;
 
+div.date {
+  @include clearfix;
+  margin-bottom: 12px;
+  .header {
+    margin-bottom: 12px;
+    .permalink {
```

Comment:
Rule declaration should be preceded by an empty line<br>Selector should have depth of applicability no greater than 2, but was 3